packages/geo/src/lib/search/search-bar/search-bar.component.ts

```diff
@@ -476,4 +476,33 @@ export class SearchBarComponent implements OnInit, OnDestroy {
       this.store.updateMany(newResults);
     }
   }
+
+  /**
+   * When the user clicks on the magnifying glass and
+   * this find the first object on the map otherwise the
+   * coordinate it will be the second option to be focused in the map
+   */
+
+ selectFirstElement(){
+
+     //Find the max value of scores
+      const maxScore = Math.max(...this.store.all().map(result => result.meta.score));
+
+      //Filter values who have the maxScore
+      const result = this.store.all().find(result => result.meta.score === maxScore);
+      //If the value has not a first maxScore it has to take the title
+      const coordInv=this.store.all().find(result => result.meta.title);
+
+      //Function to reverse the value of the parameter
+      function reverseString(coordReInv) {
+        return coordReInv;
```

Comment:
The function returns coordReInv which is what is provided to the function. Necessary?

Reply:
Yes, because the changes on design tool makes a slight change to the coordinate order. For this reason, when a search was performed, it was not executed. By reversing the coordinate values IGO can clearly understand that they are coordinates and focus them properly as desired.